tools/cockpit.spec

```diff
@@ -257,12 +278,21 @@ rm -f %{buildroot}/%{_prefix}/share/metainfo/org.cockpit-project.cockpit-docker.
 
 sed -i "s|%{buildroot}||" *.list
 
+%if 0%{?suse_version}
+# remove other brandings as they contain stale symlinks
+pushd %{buildroot}/%{_datadir}/cockpit/branding
+ls -1 | grep -v default | xargs rm -vr
+popd
```

Comment:
There's no way to override the stale symlinks check? The branding is useful for connecting to remote machines, which may run such a different OS, like Debian or Fedora.

Also, at some point you might want to add some OpenSUSE branding.

But that's not a veto from me, just a thought.

Reply:
I'll have to check. When connecting to a remote system would the branding be fetched from that remote system too? If so then the branding in the webui host wouldn't matter. If not then the branding would be broken as on SUSE systems those Fedora branding packages are missing and vice versa. The package build would have bundle a copy then.

Reply:
Sorry, indeed that is not for remote machines,  but for using the cockpit/ws container. That's not relevant for the OpenSUSE  rpms (unless you want to provide a SUSE rebuild of the container, but there's not much point).  So ok to remove the branding.